bridgestyle/arcgis/togeostyler.py

```diff
@@ -519,6 +523,8 @@ def processSymbolLayer(layer, symboltype, options):
             }
             if "dasharray" in effects:
                 stroke["dasharray"] = effects["dasharray"]
+            if "offset" in effects:
+                stroke["perpendicularOffset"] = effects["offset"] * -1
```

Comment:
Is it not a bit strange to have the `* -1` here ? Is it not better to have it in the process effect function ?

Reply:
I also wondered about this. I figured here is where we convert the lyrx object ("effects") to the GeoStyler one ("stroke") so it made some sense.
Care to expand a bit on why you think that's strange / why it would be better in the function?

Reply:
Well, for me if we have a function `processEffect`, I expect that the result is as final as possible.
Imagine we use a second time this function elsewhere, I won't add it again this * -1.
If the offset is always the invert of original value, let's do that as soon as possible in the code.

But that's just a small remark, without knowing more about the context. Feel free to merge this PR with or without addressing this comment.

Reply:
Good point, I'll make the change